settings.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const sourceMapSupport = require('source-map-support')
+const settings = require('libtap/settings');
+
+sourceMapSupport.install({environment:'node', hookRequire: true})
+
+if (+process.env.TAP_DEV_LONGSTACK !== 1) {
+  settings.stackUtils.ignoredPackages.push(
+    'libtap',
+    'tap',
+    'esm',
+    'nyc',
+    'import-jsx',
+    'function-loop'
+  )
+  settings.stackUtils.internals.push(
+    /at Generator\.next \(<anonymous>\)/iu
+  )
+} else {
+  settings.atTap = true
```

Comment:
By default `libtap` will strip `at:` if it points into `libtap` but does not add `libtap` to the ignoredPackages.  Maybe this should be tweaked so we just check for `libtap` in the ignoredPackages instead of having a separate `atTap` setting?  Maybe `libtap` could be altered to strip any `at:` which matches something in `ignoredPackages` but that might be more difficult to implement and test.